indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -421,6 +421,7 @@ public long push() throws IOException
       case "hdfs":
       case "viewfs":
       case "maprfs":
+      case "s3a":
```

Comment:
@gianm as you can see [here](https://github.com/druid-io/druid/pull/3940/files#diff-e90f94007a7b4de9213cf5c3a7ca3afdR145) i have added the hadoop aws jars to be able to read the segment with hdfs loadspec. That makes it more uniform IMO. The idea here is to have one deep storage module that read all the supported file system by hadoop.